core/api/src/services/lnd/index.ts

```diff
@@ -884,6 +910,7 @@ export const LndService = (): ILightningService | LightningServiceError => {
       listPendingPayments: listPaymentsFactory(getPendingPayments),
       listFailedPayments,
       listInvoices,
+      listAllInvoices,
```

Comment:
it's kind of confusing to have a `listInvoices` and `listAllInvoices`

Reply:
if the main reason is because of the yield, maybe `yieldInvoices` would be a better name? we could also still pass `createdAfter` to have the same signature.

Reply:
I will change the name and pass in the createdAfter

Reply:
or alternatively, just modify listInvoices in place and update get-held-invoice  and trigger to be using the `yield` interface. those seems the only 2 place there this is used and using the yield may be a few lines change max

Reply:
I changed the interface to use yield